apps/website/src/pages/about/tech/commands.tsx

```diff
@@ -220,13 +223,28 @@ const AboutTechPage: NextPage = () => {
                           <pre>
                             <code className="text-sm">
                               {signature(command)}
+                              <IconCopy
+                                onClick={() => {
+                                  copyToClipboard(signature(command));
+                                  setIndex(command.description);
+                                  setTimeout(() => {
+                                    setIndex("");
+                                  }, 2000);
+                                }}
+                              />
```

Comment:
This should be a proper `<button>`. Also does this work well if you click multiple commands fast? I think we would need to keep the timeout id somewhere and reset it when clicking to prevent timeout handlers overriding each other.

Since we have pretty much the same button again below, it's probably worth to put the button in a reusable component - can be in the same file.